app/src/app/api/v0/city/[city]/user/[user]/route.ts

```diff
@@ -18,9 +18,6 @@ export const GET = apiHandler(async (_req: NextRequest, { params }) => {
       {
         model: db.models.City,
         as: "cities",
-        where: {
-          locode: locode,
-        },
```

Comment:
Why remove this?

Reply:
didn't require this either after switching to the other suggested GET route